### DISCUSSION TITLE
How to *always* show the Python version?

### DISCUSSION THREAD

**Discussion by pzelnip** [starship/starship#2364] [Q&A]
I want my prompt to *always* display the python version in it, even if I'm not currently in a directory with a .py file, requirements.txt, etc.  I use Python extensively and am constantly switching between versions, so always want to know wherever I am in my directory structure if I type "python" and hit enter, what version will activate.

I tried:

```
[python]
detect_files = []
detect_extensions = []
```
But still it only shows when the directory I'm in meets one of the [criteria outlined in the docs](https://starship.rs/config/#python).

**Comment by edrogers** [comment]
A hacky solution that works for me:

```toml
[python]
detect_env_vars = [ "PATH" ]
```

The python version will be checked anytime the `$PATH` environment variable is set. In my shell usage, this is always true.

Hope that helps!
